test/functional/test_runner.py

```diff
@@ -384,6 +393,11 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
     if os.path.isdir(cache_dir):
         print("%sWARNING!%s There is a cache directory here: %s. If tests fail unexpectedly, try deleting the cache directory." % (BOLD[1], BOLD[0], cache_dir))
 
+    # Test Framework Tests
+    for module in TEST_FRAMEWORK_MODULES:
+        suite = unittest.TestLoader().loadTestsFromModule(module)
+        unittest.TextTestRunner(verbosity=2).run(suite)
```

Comment:
It'd be nice if we could have less verbosity for successful unittest runs, and only have verbose output for tests that fail. Otherwise, these will take up a lot of screen space for every run.

Reply:
Good point, thank you so much for all the feedback!
Verbosity=1 does the least printing, but still prints every time `TestCase.run()` is called, so I made it more concise by grouping all the tests into one `TestSuite` and calling `run()` just once instead of doing them each individually. These tests would probably all be very small so I figured it would be alright. On success, it's 4 lines. On fail, it's more verbose and quits early. Hopefully this is ok?